### PR TITLE
MajorBBS Ordinal 432

### LINKED ISSUE #146: [MJWMUT] Mutants! - Crashes when you load a registered copy of Mutants!
**Module Information**
* Identifier: [MJWMUT]
* Module Name: Mutants!
* Version: 3.11

**Describe the bug**
Crashes when you load a registered copy of Mutants!

**To Reproduce**
1. Add Mutants to moduleConfig.json and valid reg code to .msg file
2. Load MBBSEmu
3. See error

**Expected behavior**
Game loads normally

**Screenshots**
```
Critical Exception has occurred:
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter 'Unknown Exported Function Ordinal in MAJORBBS: 432')
   at MBBSEmu.HostProcess.ExportedModules.Majorbbs.Invoke(UInt16 ordinal, Boolean offsetsOnly)
   at MBBSEmu.HostProcess.ExecutionUnits.ExecutionUnit.ExternalFunctionDelegate(UInt16 ordinal, UInt16 functionOrdinal)
   at MBBSEmu.HostProcess.ExecutionUnits.ExecutionUnit.Execute(IntPtr16 entryPoint, UInt16 channelNumber, Boolean simulateCallFar, Boolean bypassState, Queue`1 initialStackValues, UInt16 initialStackPointer)
   at MBBSEmu.Module.MbbsModule.Execute(IntPtr16 entryPoint, UInt16 channelNumber, Boolean simulateCallFar, Boolean bypassSetState, Queue`1 initialStackValues, UInt16 initialStackPointer)
   at MBBSEmu.HostProcess.MbbsHost.AddModule(MbbsModule module)
   at MBBSEmu.Program.Run(String[] args)
```

**Software Information:**
- OS: Windows Server 2019 Standard
- Telnet Client: Netrunner

### PR DESCRIPTION
**Pull request by tuday2**
Closes #146 
Added MajorBBS Ordinal 432 & Test

### CODE REVIEW

File: MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs [hunk 1]
```diff
@@ -7069,6 +7072,17 @@ private void uidkey()
...
         }
 
+        /// <summary>
+        ///     Create a new key record
+        ///
+        ///     Signature: int nkyrec (char *uid);
+        /// </summary>
+        private void nkyrec()
+        {
+            // no key support for now, so everybody has the key
```
Comment (enusbaum) [inline MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs:7082]: Let's add a debug here that a new key record is being created and write out the value passed in. It'll help if this stubbed out version doesn't work for a module, well at least have the debug that the api is being called.

**Review by enusbaum — CHANGES_REQUESTED** [review]
Since it's stubbed, just adding some debug logging there to know it's being called

File: MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs (hunk 1 above)
Comment (tuday2) [reply MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs:7089]: Debug added

**Comment by tuday2** [comment]
Updated to string

**Comment by tuday2** [comment]
Return Pointer.

**Comment by tuday2** [comment]
Updated based on IDA output.

**Comment by tuday2** [comment]
Comment fixed.

**Review by enusbaum — APPROVED** [review]

**Pull request merged**